vxquery-rest/src/test/resources/vxquery.properties

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Having a properties file is good, but it should be located in vxquery-server. I think this is an issue we will have to address later when we update the vxquery-server to use NCService.